### PR TITLE
[CITE-57] Added full author name submitted to Zotero.

### PR DESCRIPTION
**Pull request by namrathaov**

### CODE REVIEW

File: citesphere-importer/src/main/java/edu/asu/diging/citesphere/importer/core/service/parse/wos/tagged/CreatorTagHandler.java [hunk 1]
```diff
@@ -31,6 +31,8 @@ protected void addFullnameToContributor(String nameString, int fieldIdx, List<Co
         // we assume that the order of standardized names is the same as of fullnames
         if (contributors.size() > fieldIdx) {
             Contributor c = contributors.get(fieldIdx);
+            c.setGivenName(name.givenName);
+            c.setSurname(name.surname);
```
Comment (jdamerow) [inline citesphere-importer/src/main/java/edu/asu/diging/citesphere/importer/core/service/parse/wos/tagged/CreatorTagHandler.java]: no that would override the short version of the name. I think the full name is simply not added to the extra field when submitted to zotero.

**Comment by namrathaov** [comment]
Added changes to submit full name to Zotero.

File: citesphere-importer/src/main/java/edu/asu/diging/citesphere/importer/core/zotero/template/impl/ExtraFieldHelper.java [hunk 2]
```diff
@@ -433,9 +433,29 @@ private void fillPerson(Contributor contrib, ObjectNode creatorNode, int idx) {
...
+            creatorNode.put("name", String.join(" ", nameParts));
+            creatorNode.put("firstName", contrib.getGivenName());
+            creatorNode.put("lastName", contrib.getSurname());
+        }
+        creatorNode.put("fullName", String.join(" ", fullNameParts));
+        creatorNode.put("fullFirstName", contrib.getFullGivenName());
+        creatorNode.put("fullLastName", contrib.getFullSurname());
+        creatorNode.put("standardizeName", String.join(" ", nameParts));
+        creatorNode.put("standardizeFirstName", contrib.getGivenName());
+        creatorNode.put("standardizeLastName", contrib.getSurname());
```
Comment (jdamerow) [inline citesphere-importer/src/main/java/edu/asu/diging/citesphere/importer/core/zotero/template/impl/ExtraFieldHelper.java]: should be 'standardize**d**' not 'standardize'

**Comment by jdamerow** [comment]
Jenkins, test this please.

**Comment by jdamerow** [comment]
After this one little change, it should be good to go.

**Comment by jdamerow** [comment]
Jenkins, test this please.

**Pull request merged**